File: FantasyRadio/src/main/java/ru/sigil/fantasyradio/About.java
package ru.sigil.fantasyradio;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

/**
 * Окошко "О программе"
 */
public class About extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.about);
        TextView tv = (TextView) findViewById(R.id.about_content);
        try {
            tv.append(getString(R.string.versiontext) + " "
                    + this.getPackageManager().getPackageInfo(this.getPackageName(), 0).versionName);
        } catch (Exception e) {
            e.printStackTrace();
        }
        tv.setOnClickListener(v -> stop());
    }

    private void stop() {
        finish();
    }

    @Override
    public void onStart() {
        super.onStart();
    }

    @Override
    public void onStop() {
        super.onStop();
    }
}

File: FantasyRadio/src/main/kotlin/ru/sigil/fantasyradio/About.kt
package ru.sigil.fantasyradio

import android.app.Activity
import android.widget.TextView
import android.os.Bundle

/**
 * Окошко "О программе"
 * Created by namelessone
 * on 08.12.18.
 */
class About: Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.about)
        val tv = findViewById<TextView>(R.id.about_content)
        tv.append(getString(R.string.versiontext) + " "
                + this.packageManager.getPackageInfo(this.packageName, 0).versionName)
        tv.setOnClickListener { _ -> stop() }
    }

    private fun stop() = finish()
}